### ISSUE TITLE
Not displays in page builder

### ISSUE THREAD

**Issue by bilalmalkoc** [PrafullaKumarSahu/siteorigin-hello-world-widget#1]
Hello. This widget not displays in page builder? What is wrong?

https://s17.postimg.org/mv3zmfizj/Kayit_4_Aral_k_2017_Pazartesi_02h08m39s_003.png

**Comment by PrafullaKumarSahu** [comment]
Hello @bilalmalkoc ! Can you please share your code?

**Comment by bilalmalkoc** [comment]
Hello @PrafullaKumarSahu 
I just installed and activated this plugin:
[https://github.com/PrafullaKumarSahu/siteorigin-hello-world-widget](url)

**Comment by PrafullaKumarSahu** [comment]
I have not used this from a long time, I will check it for you, please let me know your wordpress version and php version.

**Comment by bilalmalkoc** [comment]
I am using php 5.6.30 and wordpress 4.8.4
@PrafullaKumarSahu

**Comment by PrafullaKumarSahu** [comment]
In my installation PHP  7.0 and wordpress 4.9.1, I can not see these widgets, can you confirm, if these widgets are activated?

**Comment by bilalmalkoc** [comment]
How can i confirm it? Can you explain me?

**Comment by PrafullaKumarSahu** [comment]
Go to plugins > on hover you can see "Siteorigin Widgets", here you can see if those are active or inactive.

**Comment by bilalmalkoc** [comment]
Yes ofc. I activadet all.
http://prntscr.com/hjj568

**Comment by PrafullaKumarSahu** [comment]
I want you look into this

![siteorigin widgets](https://user-images.githubusercontent.com/2470646/33626163-c53c6542-da1f-11e7-8a02-da7cabf89dc1.png)

**Comment by PrafullaKumarSahu** [comment]
You need to check here

![widgets](https://user-images.githubusercontent.com/2470646/33626300-324a1b84-da20-11e7-8951-d6d190433cf6.PNG)

**Comment by bilalmalkoc** [comment]
I don't have it sory.
http://prntscr.com/hjj8jj

**Comment by PrafullaKumarSahu** [comment]
Ok, intall SiteOrigin CSS and SiteOrigin Widgets Bundle plugins and let me know the new status.

**Comment by bilalmalkoc** [comment]
Ok. I can see now. Thank you so much :)

**Comment by PrafullaKumarSahu** [comment]
Your issue resolved?

**Comment by bilalmalkoc** [comment]
Yes resolved. But i wonder, is possible to make it without "SiteOrigin Widgets Bundle" plugin?

**Comment by PrafullaKumarSahu** [comment]
May be I need to check, I have forgotten these things :D let me check it.

**Comment by bilalmalkoc** [comment]
Ok. Thank you :)

**Comment by PrafullaKumarSahu** [comment]
I think you can activate these widgets with some code, using an action hook and use it with out "SiteOrigin Widgets Bundle".

**Comment by PrafullaKumarSahu** [comment]
Check if these widgets are available in wordpess > appearance > widgets section if widgets are available there I think you can use them in page builder.

**Comment by bilalmalkoc** [comment]
Yes there all available in widgets section. But they available with "SiteOrigin Widgets Bundle" only.

**Comment by PrafullaKumarSahu** [comment]
Okay, in weekend I will try to look into it, if you are working on it and find any idea you can post here. By the way what you are trying to achieve?

**Comment by bilalmalkoc** [comment]
Thank you. I make new wordpress theme for me. But when i make a change, i must edit php files. And it is difficult. So i want to integrate siteorigin :) But i don't know how to create any custom widget. I will learn with your simple hello world widget.

**Comment by PrafullaKumarSahu** [comment]
Do you understand how this hello world is working?

**Comment by bilalmalkoc** [comment]
Yes. I created my custom widget before. But it not displayed in page builder. I will check your widget if it works. And i will compare what are different.

**Comment by PrafullaKumarSahu** [comment]
Okay. Good luck to you.

**Comment by bilalmalkoc** [comment]
Thanks a lot. I will wait you.

**Comment by PrafullaKumarSahu** [comment]
Okay.

**Comment by PrafullaKumarSahu** [comment]
Hi @bilalmalkoc I did not get time to look into the codes but from documentation I do not see any hook described to active the widgets without widget-bundle, but I believe you can get that code from the widget bundle and use it in your theme. I would also like to suggest you to look into this [theme](https://github.com/siteorigin/siteorigin-corp) may be this will give you some better idea for your theme, I have not tested it if it has implemented widgets the way you want but would be helpful to develop your theme.
